core/src/main/java/org/jdbi/v3/core/argument/ObjectMethodArguments.java

```diff
@@ -50,6 +49,20 @@ public ObjectMethodArguments(String prefix, Object object) {
         this.methods = CLASS_METHODS.get(object.getClass());
     }
 
+    private static Map<String, Method> load(Class<?> type) {
+        // The package check could be removed if jdbi's tests would only use public classes for argument testing
+        if (Modifier.isPublic(type.getModifiers()) || type.getPackage().equals(ObjectMethodArguments.class.getPackage())) {
```

Comment:
I'm a little worried about shipping code that behaves differently in tests explicitly.  It means we don't test the code our users run!  I think we should update all the tests to not rely on package-specific behavior, unless there's a particular reason you did it this way?

Reply:
I could remove the package check, since it is only used in some tests, but all branches of the code are tested, not just one, the package check is nessesary for some tests, but I could change them.